kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/PodOperationContext.java

```diff
@@ -143,7 +144,8 @@ public String getLogParameters() {
     if (sinceSeconds != null) {
       sb.append("&sinceSeconds=").append(sinceSeconds);
     } else if (sinceTimestamp != null) {
-      sb.append("&sinceTime=").append(sinceTimestamp);
+      // https://github.com/fabric8io/kubernetes-client/issues/6459
+      sb.append("&sinceTime=").append(URLUtils.encodeToUTF(sinceTimestamp).replace("%3A", ":"));
```

Comment:
there's no reason for encoded timestamps to ever contain the `%%3A` substring, so this should be correct, and it improves readability of the encoded result, so ok with me